scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-1a.yaml

```diff
@@ -112,7 +112,7 @@
           sleep 100 # time for the admin VM to boot
 
           # wipe out shared NFS that should be used by this deployment
-          ssh root@10.162.26.129 "/var/qa-scripts/wipe_nfs_shares.sh qa$hw_number"
+          ssh root@localhost "wipe_nfs_shares qa$hw_number"
```

Comment:
The shell part of the jenkins job is run under `jenkins` user and the action above needs to be run as root on the gate. We already have ssh key auth configured, so it's practical to make use of it.